cmd/k8s-operator/deploy/crds/tailscale.com_recorders.yaml

```
@@ -1557,6 +1557,34 @@ spec:
                                     May also be set in PodSecurityContext. If set in both SecurityContext and
                                     PodSecurityContext, the value specified in SecurityContext takes precedence.
                                   type: string
+                        serviceAccount:
+                          description: |-
+                            Config for the service account to create for the Recorder's StatefulSet.
```

Nit:
```suggestion
Config for the ServiceAccount to create for the Recorder's StatefulSet.
```

(for consistency with `StatefulSet`)